I think I have a fix for this warning:

```
  CC       plugins/src_connmand-arfailure.o
plugins/arfailure.c: In function 'timer_callback':
plugins/arfailure.c:49:2: warning: implicit declaration of function '__connman_service_clear_error' [-Wimplicit-function-declaration]
  __connman_service_clear_error(service);
  ^
plugins/arfailure.c:51:2: warning: implicit declaration of function '__connman_service_auto_connect' [-Wimplicit-function-declaration]
  __connman_service_auto_connect(CONNMAN_SERVICE_CONNECT_REASON_AUTO);
  ^
plugins/arfailure.c: In function 'service_state_changed':
plugins/arfailure.c:79:4: warning: implicit declaration of function '__connman_service_get_favorite' [-Wimplicit-function-declaration]
    __connman_service_get_favorite(service)) {
    ^
plugins/arfailure.c: In function 'arfailure_init':
plugins/arfailure.c:101:2: warning: implicit declaration of function 'connman_auto_reset_failure_timeout' [-Wimplicit-function-declaration]
  timeout = connman_auto_reset_failure_timeout();
  ^
```

--- a/bootstrap-configure-mf
+++ b/bootstrap-configure-mf
@@ -10,7 +10,7 @@ fi
   --mandir=/usr/share/man \
   --localstatedir=/var \
   --sysconfdir=/etc \
-  --enable-arfailure=builtin \
+  --enable-arfailure \
   --enable-openconnect=builtin \
   --enable-openvpn=builtin \
   --enable-vpnc=builtin \

--- a/docker_build.sh
+++ b/docker_build.sh
@@ -3,7 +3,7 @@
 set -eux
 
 # Debian package metadata
-PKG_NAME="connman"
+PKG_NAME="connman-mf"
 SRC_VERSION="1.33"
 PKG_RELEASE="0"
 

--- a/plugins/arfailure.c
+++ b/plugins/arfailure.c
@@ -24,10 +24,12 @@
 #endif
 
 #define CONNMAN_API_SUBJECT_TO_CHANGE
+#include "connman.h"
 #include <connman/plugin.h>
 #include <connman/log.h>
 #include <connman/notifier.h>
 #include <connman/service.h>
+#include <connman/setting.h>
 
 #include <glib.h>
 

